### PR TITLE
Manage Menu:Adding Three Dots on Manage Menu

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

**Comment by dannyparsons** [comment]
Can you include all the menu changes asked by Roger (highlighted yellow in his email yesterday) in this pull request? There will be less likely conflicts if you do it all at once. Thanks

**Comment by Lunalo** [comment]
Hi Danny, Am now Done with all Changes on the form Main

**Comment by dannyparsons** [comment]
I have sent an email with corrects for this.

**Comment by Lunalo** [comment]
Hi Danny,

My visual studio crashed this evening.

I request you merge those changes  and then it will be easier for me
to make those changes.



That task involving  view metadata needs your help. I was supposed to
Skype you but there was no enough internet.
Thanks

On 2/15/16, Danny Parsons <notifications@github.com> wrote:
> I have sent an email with corrects for this.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/africanmathsinitiative/Instat/pull/809#issuecomment-184346312


-- 
Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

**Comment by dannyparsons** [comment]
Sorry to hear that. Well done for committing your changes then so that they were not lost when your machine crashed! Is your Visual Studio working now? If so you can just continue on this branch.

**Comment by Lunalo** [comment]
I have managed to fix and I can now clone and do all those changes now.

On 2/15/16, Danny Parsons <notifications@github.com> wrote:
> Sorry to hear that. Well done for committing your changes then so that they
> were not lost when your machine crashed! Is your Visual Studio working now?
> If so you can just continue on this branch.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/africanmathsinitiative/Instat/pull/809#issuecomment-184358265


-- 
Regards,
John Ong'ala Lunalo|| Kenyatta University Alumnus||
Volunteer, African Maths Initiative||
Email Address :johnlunalo95@gmail.com
Tel: +254701519572, +254786438572

File: instat/frmMain.Designer.vb [hunk 1]
```diff
@@ -1826,156 +1913,177 @@ Partial Class frmMain
...
+        Me.mnuManageDataFrameDisplay.Text = "Display..."
         '
         'mnuManageDataFrameProtectLockUnlock
         '
         Me.mnuManageDataFrameProtectLockUnlock.Name = "mnuManageDataFrameProtectLockUnlock"
-        Me.mnuManageDataFrameProtectLockUnlock.Size = New System.Drawing.Size(183, 22)
+        Me.mnuManageDataFrameProtectLockUnlock.Size = New System.Drawing.Size(207, 22)
         Me.mnuManageDataFrameProtectLockUnlock.Tag = "Protect_Lock_Unlock"
-        Me.mnuManageDataFrameProtectLockUnlock.Text = "Protect(Lock/Unlock"
+        Me.mnuManageDataFrameProtectLockUnlock.Text = "Protect(Lock/Unlock)..."
```
Comment (dannyparsons) [inline instat/frmMain.Designer.vb]: there should be a space here after Protect.

File: instat/frmMain.Designer.vb [hunk 2]
```diff
@@ -1826,156 +1913,177 @@ Partial Class frmMain
...
+        Me.mnuManageDataFile.Size = New System.Drawing.Size(152, 22)
         Me.mnuManageDataFile.Tag = "Data_file"
         Me.mnuManageDataFile.Text = "Data File"
         '
+        'mnuManageDataFileViewDataMetadata
+        '
+        Me.mnuManageDataFileViewDataMetadata.Name = "mnuManageDataFileViewDataMetadata"
+        Me.mnuManageDataFileViewDataMetadata.Size = New System.Drawing.Size(188, 22)
+        Me.mnuManageDataFileViewDataMetadata.Tag = "View_Data_Metadata"
+        Me.mnuManageDataFileViewDataMetadata.Text = "View Data Metadata..."
```
Comment (dannyparsons) [inline instat/frmMain.Designer.vb]: This should be "View Data Frame Metadata"

**Pull request merged**